chat/chat-api/api/src/java/org/sakaiproject/chat2/model/MessageDateString.java

```diff
@@ -0,0 +1,12 @@
+package org.sakaiproject.chat2.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class MessageDateString{
+	private String pdate;
+	private String ptime;
+	private String pid;
+}
```

Comment:
I don't know... these are the same names as used in the old chat. Maybe we can change them for  more descriptive ones.